src/client.js

```diff
@@ -3020,6 +3020,25 @@ module.exports.CRYPTO_ENABLED = CRYPTO_ENABLED;
  */
 
 /**
+ * Fires whenever an error occurs when call.js sendEvent encounters an error.
+ * <p>
+ * This is required to allow UnknownDeviceError to propagate from client.sendEvent, whilst
+ * differentiating it from being a normal `error` emission. (This is because call.js
+ * does a hangup when it encounters a normal `error`, which in turn could lead to an
+ * UnknownDeviceError.)
+ * <p>
+ * To deal with an UnknownDeviceError when trying to send events, the application should let
+ * users know that there are new devices in the encrypted room (into which the event was
+ * sent) and give the user the options to resend unsent events or cancel them.
+ *
```

Comment:
worth noting what the MatrixCall does. Does it hang up? does it carry on regardless? does it block until you resend the event?